modules/chargebee-integration/app/kuberlogic.go

```diff
@@ -41,7 +44,124 @@ func createService(logger *zap.SugaredLogger, svc *models.Service) error {
 	return nil
 }
 
+func deleteService(logger *zap.SugaredLogger, serviceId string) error {
+	params := service.NewServiceDeleteParams()
+	params.ServiceID = serviceId
+
+	apiClient, err := makeClient()
+	if err != nil {
+		return err
+	}
+	_, err = apiClient.Service.ServiceDelete(
+		params,
+		httptransport.APIKeyAuth("X-Token", "header", viper.GetString("KUBERLOGIC_APISERVER_TOKEN")),
+	)
+	return nil
+}
+
+func getServiceBySubscriptionId(logger *zap.SugaredLogger, subscriptionId string) (*models.Service, error) {
+	params := service.NewServiceListParams()
+	params.SubscriptionID = &subscriptionId
+
+	apiClient, err := makeClient()
+	if err != nil {
+		return nil, err
+	}
+
+	response, err := apiClient.Service.ServiceList(
+		params,
+		httptransport.APIKeyAuth("X-Token", "header", viper.GetString("KUBERLOGIC_APISERVER_TOKEN")),
+	)
+	if err != nil {
+		return nil, err
+	}
+	if len(response.Payload) > 0 {
```

Comment:
It's worth to fail when we have more than one service

Reply:
We can have multiple services with single subscription id?